packages/react-client/src/__tests__/ReactFlight-test.js

```diff
@@ -68,10 +68,15 @@ let ErrorBoundary;
 let NoErrorExpected;
 let Scheduler;
 let assertLog;
+let assertConsoleErrorDev;
 
 describe('ReactFlight', () => {
   beforeEach(() => {
     jest.resetModules();
+    console.createTask = function fakeCreateTask(name) {
+      return {run: fn => fn()};
+    };
```

Comment:
We should think through a way to test these properly which might be dependent on what the public API for getting the owner stacks is.

We could maybe also polyfill it with by appending to console.error so that our existing tests pass.

Reply:
Yeah, this was a quick mock to reveal the error. Once we land https://github.com/facebook/react/pull/29762, I'll clean this up.